cloud/pkg/edgecontroller/messagelayer/util.go

```diff
@@ -25,6 +25,16 @@ func BuildResource(nodeID, namespace, resourceType, resourceID string) (resource
 	return
 }
 
+// BuildResourceForRouter return a string as "beehive/pkg/core/model".Message.Router.Resource
+func BuildResourceForRouter(resourceType, resourceID string) (resource string, err error) {
```

Comment:
Use explicit return, do not name return value.

Reply:
done